samples/bot-adaptive-card-actions/csharp/AdaptiveCardActions/Bots/AdaptiveActionsBot.cs

```diff
@@ -0,0 +1,161 @@
+                }
+            }
+
+            await SendDataOnCardActions(turnContext, cancellationToken);
+        }
+
+        private static string ProcessInput(string text)
+        {
+            const string colorText = "is the best color, I agree.";
+            switch (text)
+            {
+                case "red":
+                    {
+                        return $"Red {colorText}";
+                    }
+                case "yellow":
+                    {
+                        return $"Yellow {colorText}";
+                    }
+                case "blue":
+                    {
+                        return $"Blue {colorText}";
+                    }
+                default:
+                    {
+                        return "Please select a color from the suggested action choices";
+                    }
+            }
+        }
+
+        // Creates and sends an activity with suggested actions to the user. When the user
+        /// clicks one of the buttons the text value from the "CardAction" will be
+        /// displayed in the channel just as if the user entered the text. There are multiple
+        /// "ActionTypes" that may be used for different situations.
+        private static async Task SendSuggestedActionsAsync(ITurnContext turnContext, CancellationToken cancellationToken)
+        {
+            try
+            {
+                var reply = MessageFactory.Text("What is your favorite color?");
+                reply.SuggestedActions = new SuggestedActions()
+                {
+                    Actions = new List<CardAction>()
+                {
+                    new CardAction() { Title = "Red", Type = ActionTypes.ImBack, Value = "Red" },
+                    new CardAction() { Title = "Yellow", Type = ActionTypes.ImBack, Value = "Yellow" },
+                    new CardAction() { Title = "Blue", Type = ActionTypes.ImBack, Value = "Blue" },
+                },
+                    To = new List<string> { turnContext.Activity.From.Id },
+                };
+
+                await turnContext.SendActivityAsync(reply, cancellationToken);
+            }
+            catch (Exception e)
+            {
+                throw (e);
+            }
+        }
+
+        // Get text.input submitted value from card.
+        private async Task SendDataOnCardActions(ITurnContext turnContext, CancellationToken cancellationToken)
+        {
+            if (turnContext.Activity.Value != null)
+            {
+                var reply = MessageFactory.Text("");
+                reply.Text = $"Data Submitted : {turnContext.Activity.Value}";
+                await turnContext.SendActivityAsync(MessageFactory.Text(reply.Text), cancellationToken);
+            }
+        }
+
+        // Get intial card.
+        private Attachment GetFirstOptionsAdaptiveCard(string[] filepath, string name = null, string userMRI = null)
```

Comment:
This summary is not correct.
Add proper summary with ///

Reply:
fixed